src/components/ui/toast.tsx

```diff
@@ -0,0 +1,115 @@
+'use client';
+
+import * as React from 'react';
+import * as ToastPrimitives from '@radix-ui/react-toast';
+import { cva, type VariantProps } from 'class-variance-authority';
+import { X } from 'lucide-react';
+
+import { cn } from '@/lib/utils';
+
+const ToastProvider = ToastPrimitives.Provider;
+
+const ToastViewport = React.forwardRef<
+  React.ElementRef<typeof ToastPrimitives.Viewport>,
+  React.ComponentPropsWithoutRef<typeof ToastPrimitives.Viewport>
+>(({ className, ...props }, ref) => (
+  <ToastPrimitives.Viewport
+    ref={ref}
+    className={cn(
+      'fixed top-0 z-[100] flex max-h-screen w-full flex-col-reverse px-5 sm:bottom-0 sm:right-0 sm:top-auto sm:flex-col md:max-w-[420px] text-destructive',
```

Comment:
Should we always make the toast appear from the top regardless of the screen size? By default, it appears on the bottom right corner when it's a bigger screen. What do you think?

Reply:
I thought we don't need to consider how to display by device, so I left it as the default. In my opinion, I think it's fine not to change it at this point.

Reply:
Yes. However, I still think the toast should show up the same way on PCs and smartphones. We only care about the mobile users, but it doesn't mean the toast can appear differently.

Reply:
What kind of design do you need me to do?

Reply:
I standardized how to display the toast component.